Automattic/wp-calypso
# Add Jetpack Starter to bundle comparison table

Related to 1198218726984184-as-1204562517297149

## Proposed Changes

* Replace VaultPress Backup with Jetpack Starter on the feature comparison page, so users can compare bundles

## Testing Instructions



* Open the Jetpack Cloud live link
* Visit /features/comparison
* Confirm that Starter is on that page and it has the correct features list
* Make sure the product lightbox (after clicking on "More about Starter") is visible and shows the correct features

<img width="1285" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/9ceb9c94-2d31-4e85-9474-25c0215aad26">

## Pre-merge Checklist
